apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new-card.js

```diff
@@ -0,0 +1,210 @@
+/*** THIS MUST BE THE FIRST THING EVALUATED IN THIS SCRIPT *****/
+// import './public-path';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PaginationControl from './pagination';
+import minimize from './icons/minimize';
+import thumbsUp from './icons/thumbs_up';
+import thumbsDown from './icons/thumbs_down';
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { Button, Card, CardBody, CardFooter, CardMedia, Flex } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+import { useEffect } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { __ } from '@wordpress/i18n';
+
+// eslint-disable-next-line react-hooks/exhaustive-deps
+const useEffectOnlyOnce = ( func ) => useEffect( func, [] );
+
+function WhatsNewCard( {
+	cardContent,
+	cardIndex,
+	justMaximized,
+	lastCardIndex,
+	onMinimize,
+	onDismiss,
+	setJustMaximized,
+	setCurrentCardIndex,
+} ) {
+	const { description, heading, imgSrc } = cardContent;
+	const isLastCard = cardIndex === lastCardIndex;
+	// Ensure tracking is recorded once per slide view
+	useEffectOnlyOnce( () => {
+		// Don't track slide view if returning from minimized state
+		if ( justMaximized ) {
+			setJustMaximized( false );
+			return;
+		}
+		recordTracksEvent( 'calypso_editor_whats_new_slide_view', {
+			slide_number: cardIndex + 1,
+			is_last_slide: isLastCard,
+			slide_heading: heading,
+			is_gutenboarding: window.calypsoifyGutenberg?.isGutenboarding,
+		} );
+	} );
+
+	return (
+		<Card className="whats-new-card" isElevated>
+			<CardOverlayControls
+				setJustMaximized={ setJustMaximized }
+				onDismiss={ onDismiss }
+				onMinimize={ onMinimize }
+				slideNumber={ cardIndex + 1 }
+			/>
+			<CardMedia>
+				<img alt={ __( "What's New", 'full-site-editing' ) } src={ imgSrc } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'What's new' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/whats-new)